lighthouse-core/runner.js

```diff
@@ -269,8 +275,7 @@ class Runner {
       const auditOptions = Object.assign({}, audit.defaultOptions, auditDefn.options);
       const auditContext = {
         options: auditOptions,
-        settings,
-        LighthouseRunWarnings: runWarnings,
+        ...auditsContext,
```

Comment:
dont love these two diff objs with such similar names
how about `sharedAuditsContext` for this one?

Reply:
sure

Reply:
done